_posts/2021-02-09-newpipe-0.20.9-released.md

```diff
@@ -0,0 +1,124 @@
+excerpt_separator: <!-- more -->
+---
+
+Long time no see, hmm. We were busy behind the scenes and can present you quite some new features as well as a lot of bug fixes and improvements. First and foremost though, two more developers have joined Team NewPipe. Welcome, [@XiangRongLin](https://github.com/XiangRongLin) and [@vkay94](https://github.com/vkay94)!
+
+<!-- more -->
+
+## New
+
+### PeerTube Sepia Search
+PeerTube has had a longstanding problem that it is difficult to search for videos _across_ instances when they don't federate, i.e., list videos from other instances. So Framasoft developed Sepia Search, a new search engine for PeerTube. You can read more about it over at the [Framasoft blog](https://framablog.org/2020/09/22/sepia-search-our-search-engine-to-promote-peertube/). Now you can use this search engine within NewPipe as well, thanks to [@Bopol](https://github.com/b0pol). You can select it from the three dot menu on the right side of the search bar (just like YouTube Music) when you are on a PeerTube instance, and you are good to search all the instances available [here](https://instances.joinpeertube.org/instances)!
+
+### YouTube Chapters
+YouTube introduced Chapters in May last year, allowing viewers to easily jump to specific timestamps in a video which the content creator picked for them. With this release, [@vkay94](https://github.com/vkay94) brought this feature to you, the users of NewPipe, as well. You can select chapters by pressing the new Chapters button in the video player, whose icon looks like a numbered list (as opposed to the Playlist button whose icon is a bulleted list). The list of chapters looks like this:
+
+<img class="no-flow vertical" src="{{ site.baseurl }}/img/screenshots/chapters.png"/>
+
+### Bottom Tabs and Expanded Buttons (the Share button is back, baby!)
+We heard you like buttons (quite loud and clear), so we brought you more. Rather, [@Stypox](https://github.com/Stypox) restored a couple of buttons which didn't find a place in the transition to the unified UI. NewPipe now shows more buttons below the visible button row by using the expansion arrow. You can now directly share the video you are watching to other apps or open it in your browser. If you enable the "Show Kodi" Setting, you can also play the video with Kodi directly from there.
+
+Along with this, [@Stypox](https://github.com/Stypox) also added a tab layout for the components below the video. Now you have three tabs you can tap to switch between Comments, Related Videos and the Video Description. This looks a little something like:
+
+<img class="no-flow img-responsive" src="{{ site.baseurl }}/img/screenshots/buttons.png"/>
+
+
+### This is not all
+
+Another small feature found its way into this release: Channels can like comments on their videos, which results in a little heart on screen. NewPipe extracts this heart now from YouTube and displays it next to the comment. Heartfelt thanks [@B0pol](https://github.com/b0pol) for that!
```

Comment:
this is false unless I missed a PR on NewPipe, I only implemented the extractor part

Reply:
mkay